### PR TITLE
Ssl internal noacl

### PR DESCRIPTION
**Pull request by fekete-robert**
Updates using Kafka with SSL (no acl)
Requires https://github.com/banzaicloud/kafka-operator/pull/568

### CODE REVIEW

File: docs/enable-ssl.sample [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  externalListeners:
+    - type: "ssl"
+      name: "external"
+      externalStartingPort: 19090
+      containerPort: 29092
+  internalListeners:
+    - type: "ssl"
+      name: "ssl"
+      containerPort: 29092
+      usedForInnerBrokerCommunication: true
```
Comment (baluchicken) [inline docs/enable-ssl.sample:11]:
```suggestion
      - type: "ssl"
        name: "internal"
        containerPort: 29092
        usedForInnerBrokerCommunication: true
      - type: "ssl"
        name: "controller"
        containerPort: 29093
        usedForInnerBrokerCommunication: false
        usedForControllerCommunication: true
```

File: docs/headless/warning-listener-protocol.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+---
+{{< warning >}}After the cluster is created, you cannot change the way the listeners are configured. If a cluster is created with unencrypted (plain text) listener, it cannot be changed later to use SSL encrypted listeners, or the way around.{{< /warning >}}
```
Comment (baluchicken) [inline docs/headless/warning-listener-protocol.md:3]: I would rephrase this, by saying you can change it but not without outage or manual steps. The manual steps includes killing all the broker pods one by one

File: docs/ssl.md [hunk 3]
```diff
@@ -49,10 +37,10 @@ You can leave the `topicGrants` out as they will not have any effect.
 
 1. To enable ACL support for your kafka cluster, pass the following configurations along with your `brokerConfig`:
 
-```yaml
-authorizer.class.name=kafka.security.auth.SimpleAclAuthorizer
-allow.everyone.if.no.acl.found=false
-```
+    ```yaml
+    authorizer.class.name=kafka.security.auth.SimpleAclAuthorizer
```
Comment (baluchicken) [inline docs/ssl.md:41]:
```suggestion
    authorizer.class.name=kafka.security.authorizer.AclAuthorizer
```

File: docs/test.md [hunk 4]
```diff
@@ -99,40 +100,64 @@ You can use the following procedure to send and receive messages within a Kubern
...
+        ```
 
-- Consume messages
+    - If you have the Java Kafka client installed, run:
 
-    ```bash
-    kafka-console-consumer.sh --bootstrap-server $SERVICE_IP:$SERVICE_PORT --topic my-topic --from-beginning
-    ```
+        ```bash
+        kafka-console-producer.sh --broker-list $SERVICE_IP:$SERVICE_PORT --topic my-topic
```
Comment (baluchicken) [inline docs/test.md:139]:
```suggestion
        kafka-console-producer.sh --bootstrap-server $SERVICE_IP:$SERVICE_PORT --topic my-topic
```

File: docs/headless/warning-listener-protocol.md (hunk 2 above)
Comment (fekete-robert) [reply docs/headless/warning-listener-protocol.md:3]:
```suggestion
{{< warning >}}After the cluster is created, you cannot change the way the listeners are configured without an outage. If a cluster is created with unencrypted (plain text) listener and you want to switch to SSL encrypted listeners (or the way around), you must manually delete each broker pod. The operator will restart the pods with the new listener configuration.{{< /warning >}}
```

**Review by baluchicken — APPROVED** [review]
